pkg/transformer/openshift/openshift_test.go

```diff
@@ -243,24 +243,25 @@ func TestGetAbsBuildContext(t *testing.T) {
 	gitDir := testutils.CreateLocalGitDirectory(t)
 	testutils.SetGitRemote(t, gitDir, "newremote", "https://git.test.com/somerepo")
 	testutils.CreateGitRemoteBranch(t, gitDir, "newbranch", "newremote")
-	testutils.CreateSubdir(t, gitDir, "a/b")
+	testutils.CreateSubdir(t, gitDir, "a/b/build")
+	testutils.CreateSubdir(t, gitDir, "build")
 	dir := testutils.CreateLocalDirectory(t)
 	defer os.RemoveAll(gitDir)
 	defer os.RemoveAll(dir)
 
 	testCases := map[string]struct {
-		expectError    bool
-		context        string
-		composeFileDir string
-		output         string
+		expectError bool
+		context     string
+		output      string
 	}{
-		"Get abs build context success": {false, "./b/build", filepath.Join(gitDir, "a"), "a/b/build"},
-		"Get abs build context error":   {true, "", dir, ""},
+		"Get abs build context success case-1": {false, filepath.Join(gitDir, "a/b/build"), "a/b/build/"},
+		"Get abs build context success case-2": {false, filepath.Join(gitDir, "build"), "build/"},
+		"Get abs build context error":          {true, "example/build", "example/build/"},
```

Comment:
Since this particular test is expecting a failure so that field `output` is being used, so we can keep that string empty.